admin-dev/themes/new-theme/js/pages/currency/form/components/CurrencyFormatter.vue

```diff
@@ -28,8 +28,9 @@
       <language-list
         v-if="languagesCount"
         :languages="languages"
+        :translations="translations"
         @selectLanguage="selectLanguage"
-        @resetLanguage="resetLanguage"
+        @confirmResetLanguage="confirmResetLanguage"
```

Comment:
You don't need to change the event name in `LanguageList` The action requested by the list is still `resetLanguage` the fact that there is a confirmation is the responsibility of this component, so the callback name however makes sense